README.md

```diff
@@ -54,7 +54,7 @@ Execute installation script
      * For certify:
        * By default, the plugin will be taken from artifactory server
        * For custom plugin:
-         * In the [docker compose file](docker-compose/docker-compose-certify/docker-compose.yml) uncomment the [enable_certify_artifactory](docker-compose/docker-compose-certify/docker-compose.yml#L74) and [volume](docker-compose/docker-compose-certify/docker-compose.yml#L78)
+         * In the [docker compose file](docker-compose/docker-compose-certify/docker-compose.yml) uncomment the [enable_sunbird_certify_artifactory](docker-compose/docker-compose-certify/docker-compose.yml#L74) and [volume](docker-compose/docker-compose-certify/docker-compose.yml#L78)
```

Comment:
why is this called enable_sunbird_certify_artifactory ? the artifactory is not specific to sunbir

Reply:
changed to enable_certify_artifactory